iocBoot/iocCamera/st.cmd

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env Camera
 # -*- container-image: ghcr.io/cnpem/ad-aravis-epics-ioc
 
-cd /opt/ad-aravis-epics-ioc/iocBoot/iocCamera
-
-< envPaths
+< /usr/local/share/misc/envPaths
```

Comment:
I'm wondering if `camera-envPaths` or `envPathsCamera` wouldn't be a better name... For namespacing reasons.

(I prefer the second one)

Reply:
To namespace it, I'd rather move it to a dedicated folder under `share`. Maybe `/usr/local/share/ad-aravis{,-epics-ioc}/envPaths`. Other installed template scripts and configuration files could share the namespace this way.

But then I fear we are losing the benefit of defining a (copy-and-paste) standard for different IOCs. Do we want to namespace to support different IOCs in the same container?

Reply:
> But then I fear we are losing the benefit of defining a (copy-and-paste) standard for different IOCs. 

I don't think things are strictly copy and paste now, since the binary has a specific name...

> Do we want to namespace to support different IOCs in the same container?

For most cases, I don't think so. But I feel like using a file in the same path for every container, which has different contents based on the container, can be a bit confusing; if the path is namespaced, then it's clear it's specific to this application.

Otherwise, for consistency, there should be a dynamic link to the `Camera` binary with a generic name (and we should document such a standard). I might be in favor of it if we go the whole way, but we can discuss more in person.